packages/app/src/layouts/default.vue

```diff
@@ -36,11 +48,64 @@
 </template>
 
 <script lang="ts" setup>
+import { gql, useQuery, useMutation, useSubscription } from '@urql/vue'
 import SidebarNavigation from '../navigation/SidebarNavigation.vue'
+
 import HeaderBar from '@cy/gql-components/HeaderBar.vue'
+import BaseError from '@cy/gql-components/error/BaseError.vue'
+import Spinner from '@cy/components/Spinner.vue'
+
 import { useRoute } from 'vue-router'
 import { computed } from 'vue'
 
+import { MainAppQueryDocument, MainApp_ResetErrorsAndLoadConfigDocument, CheckBaseErrorDocument } from '../generated/graphql'
+
+gql`
+fragment MainAppQueryData on Query {
+    baseError {
+      ...BaseError
+    }
+    currentProject {
+      id
+      isLoadingConfigFile
+      isLoadingNodeEvents
+    }
+}
+`
+
+gql`
+query MainAppQuery {
+  ...MainAppQueryData
+}
+`
+
+gql`
+mutation MainApp_ResetErrorsAndLoadConfig {
+  resetErrorsAndLoadConfig {
+    ...MainAppQueryData
+  }
+}
+`
+
+gql`
+subscription CheckBaseError {
+  baseErrorChange {
+    ...MainAppQueryData
+  }
+}
+`
+
+useSubscription({ query: CheckBaseErrorDocument })
```

Comment:
Since this page will load for all routes, I think we should follow the same pattern we have for [Runner.vue](https://github.com/cypress-io/cypress/blob/b65371a6a12aa18d9cd187b5bbf9b006a49cabc7/packages/app/src/pages/Specs/Runner.vue#L69) and pause all queries/subscriptions when in run mode.

Reply:
Yeah this is a good catch.